tests/integration/cli/test_accounts.py

```diff
@@ -61,25 +65,52 @@ def test_account():
 
 def test_import(ape_cli, runner, test_account, test_keyfile_path):
     assert not test_keyfile_path.exists()
-    # Add account from private key
-    valid_input = ["0x" + PRIVATE_KEY, PASSWORD, PASSWORD]
-    result = runner.invoke(ape_cli, ["accounts", "import", ALIAS], input="\n".join(valid_input))
+    # Add account from private keys
+    result = runner.invoke(ape_cli, ["accounts", "import", ALIAS], input=IMPORT_VALID_INPUT)
     assert result.exit_code == 0
     assert test_account.address in result.output
     assert ALIAS in result.output
     assert test_keyfile_path.exists()
 
 
+def test_import_alias_already_in_use(ape_cli, runner, test_account, test_keyfile_path):
+    def invoke_import():
+        return runner.invoke(ape_cli, ["accounts", "import", ALIAS], input=IMPORT_VALID_INPUT)
+
+    result = invoke_import()
+    assert result.exit_code == 0
+    result = invoke_import()
+    assert_failure(result, f"Account with alias '{ALIAS}' is already in use")
+
+
+def test_import_account_instantiation_failure(
+    mocker, ape_cli, runner, test_account, test_keyfile_path
+):
+    eth_account_from_key_patch = mocker.patch("ape_accounts._cli.EthAccount.from_key")
```

Comment:
Using `.patch` here because the error can many different things from `EthAccount.from_key` (our dependency in this command).